### ISSUE TITLE
Using two charts in one Component: @ViewChild ref

### ISSUE THREAD

**Issue by realshaft** [emn178/angular2-chartjs#31]
Is it possible to use several chart instances in one component and reference them as

@ViewChild(ChartComponent) chart: ChartComponent;
@ViewChild(ChartComponent) chart2: ChartComponent;

or similar?

Or is it necessary to create a subcomponent for each chart I want to create?

**Comment by emn178** [comment]
Yes, you can use [Template reference variables](https://angular.io/guide/template-syntax#ref-vars).
